client-spark/common/src/test/java/org/apache/uniffle/shuffle/manager/ShuffleManagerServerFactoryTest.java

```diff
@@ -22,21 +22,16 @@
 import org.apache.uniffle.common.config.RssBaseConf;
 import org.apache.uniffle.common.rpc.ServerType;
 
-import static org.junit.jupiter.api.Assertions.assertThrows;
-
 public class ShuffleManagerServerFactoryTest {
   @Test
   public void testShuffleManagerServerType() {
```

Comment:
Could we use `@ParameterizedTest` and provide the server types as test parameters?

Reply:
Sure.